.travis.yml

```diff
@@ -5,9 +5,23 @@ php:
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
```

Comment:
Not sure if we have a consensus here yet. Do we really want to add every single future version for all components?

Other than that, LGTM :+1: